velox/docs/functions/spark/math.rst

```diff
@@ -96,6 +96,14 @@ Mathematical Functions
 
     Returns Euler's number raised to the power of ``x``.
 
+.. spark:function:: expm1(x) -> double
+
+    Returns Euler's number raised to the power of ``x`` minus 1, which is "exp(x)-1" in math, while this function is more accurate than exp(num) - 1 when the num is close to zero.
```

Comment:
add "," before "minus 1" to indicate it's not x minus 1.

Reply:
I'm confused why it is more accurate than exp(num) - 1. Could you clarify?

Reply:
I'm not so clear with the details of the implement, as I don't find the source code for the lib. But I can give the example and the explanation:
the function only provide benefits when x drops down to 0, which should be less than 1e-12 here.
I've test the 1e-12 for both cpp and java:
For cpp, the x = 1e-12, the result:
expm1(x): 1e-12
exp1(x)-1: 1.00009e-12
For java, the result:
1.0000000000005E-12
1.000088900582341E-12

you can see the difference is in the last 5 digits. So the implementation is to optimize the accuracy by some math tools like Taylor series.